src/zip.c

```diff
@@ -735,8 +734,7 @@ static int zip_central_dir_delete(mz_zip_internal_state *pState,
     d_num += end - begin;
   }
 
-  pState->m_central_dir_offsets.m_size =
```

Comment:
can you explain this change?
I would understand if we replace `mz_uint32` by `m_element_size`